realm/realm-library/src/syncIntegrationTest/java/io/realm/objectserver/ManagementRealmTests.java

```diff
@@ -137,8 +137,14 @@ public void onChange(RealmResults<PermissionOfferResponse> responses) {
                                                 // 7. Response accepted. It should now be possible for user2 to access user1's Realm
                                                 Realm realm = Realm.getInstance(config2);
                                                 looperThread.testRealms.add(realm);
-                                                assertEquals(1, realm.where(Dog.class).count());
-                                                looperThread.testComplete();
+                                                RealmResults<Dog> dogs = realm.where(Dog.class).findAll();
```

Comment:
I think you need to add this to `looperThread.strongReferences` to avoid flaky tests